src/milia/api/user.cljc

```diff
@@ -162,6 +162,10 @@
                 :http-options {:query-params {:search email}}
                 :suppress-4xx-exceptions? true)))
 
+(defn generate-odk-token []
+  (let [url (make-url "user/odk_token")]
```

Comment:
Can we refactor this to match how `make-url` has been used throughout milia and with regards to creating a url that has path separated by slashes